101-recovery-services-backup-file-share/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+    "scheduleRunTimes": {
+      "type": "array",
+      "defaultValue": [ "2020-11-04T05:30:00Z" ],
+      "metadata": {
+        "description": "Times in day when backup should be triggered. e.g. 01:00 or 13:00. Must be an array; however for File Shares only one value is valid. This will be used in LTR too for daily, weekly, monthly and yearly backup. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "timeZone": {
+      "type": "string",
+      "defaultValue": "UTC",
+      "metadata": {
+        "description": "Any valid timezone, for example: UTC, Pacific Standard Time. Refer: https://msdn.microsoft.com/en-us/library/gg154758.aspx (Ignore if using existing Backup Policy)."
+      }
+    },
+    "dailyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 30,
+      "metadata": {
+        "description": "Number of days for which the daily backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "daysOfTheWeek": {
+      "type": "array",
+      "defaultValue": [ "Sunday", "Tuesday", "Thursday" ],
+      "metadata": {
+        "description": "Array of days on which backup is to be performed for Weekly Retention. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "weeklyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 12,
+      "metadata": {
+        "description": "Number of weeks for which the weekly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "monthlyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 60,
+      "metadata": {
+        "description": "Number of months for which the monthly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "monthsOfYear": {
+      "type": "array",
+      "defaultValue": [ "January", "May", "September" ],
+      "metadata": {
+        "description": "Array of months on which backup is to be performed for Yearly Retention. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "yearlyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 10,
+      "metadata": {
+        "description": "Number of years for which the yearly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    }
+  },
+  "variables": {
+    "backupFabric": "Azure",
+    "backupManagementType": "AzureStorage",
+    "containerType": "StorageContainer",
+    "protectedItemType": "AzureFileShareProtectedItem",
+    "skuName": "RS0",
+    "skuTier": "Standard",
+    "workLoadType": "AzureFileShare"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.RecoveryServices/vaults",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('vaultName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "[variables('skuName')]",
+        "tier": "[variables('skuTier')]"
+      },
+      "properties": {},
+      "condition": "[parameters('isNewVault')]"
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
Done.